### DISCUSSION TITLE
Error with splash screen in android

### DISCUSSION THREAD

**Discussion by Rafucho25** [unoplatform/uno#15322] [Q&A]
Hi, i receive this error on Android (windows and wasm works great) every time I create a new project and try to run in debug or release mode.
![error](https://github.com/unoplatform/uno/assets/16562078/5309430e-25ed-4c46-bfd1-391522ae5743)
This happens in both Visual Studio and Visual Studio Code, with UNO platform 5 and 5.1. I've tried to clean, build and rebuild solution, specify the .net version in global.json, try to run in WSA, my Android device and emulator but nothing works. This is the last project i created, hope is useful: https://estuasdedu-my.sharepoint.com/:u:/g/personal/100401245_est_uasd_edu_do/Ed0UhiuSxaZClg5cGzDb5DUB2NIRhd0x48spk06lbnZRQg?e=AkQS4u

**Comment by jeromelaban** [comment]
Hey @pictos, is it something you've already seen?

**Comment by pictos** [comment]
Hey @Rafucho25, I tried on my end and could build with out any error... but when I try to launch your app it fails 
> [ompanyname.tes] * Assertion at /__w/1/s/src/mono/mono/metadata/object.c:657, condition `lock->done' not met
[libc] Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 20806 (.NET TP Worker), pid 20746 (ompanyname.test)
[ompanyname.tes] * Assertion at /__w/1/s/src/mono/mono/metadata/object.c:657, condition `lock->done' not met
[ompanyname.tes] * Assertion at /__w/1/s/src/mono/mono/metadata/object.c:657, condition `lock->done' not met
Loaded assembly: /data/data/com.companyname.test/files/.__override__/System.Runtime.Intrinsics.dll [External]
[monodroid-assembly] open_from_bundles: failed to load assembly System.Threading.ThreadPool.dll


Just to double check, can you run Uno.Check and make sure your env. is updated?

**Reply by jeromelaban** [reply]
The failure is caused by https://github.com/unoplatform/uno/discussions/15044, so it's unrelated.

**Reply by Rafucho25** [reply]
Hi, well in my pc i cant build, rebuild or publish any new project and this is the output for uno-check:

This tool will check your Uno Platform development environment.
If problems are detected, it will offer the option to try and fix them for you, or suggest a way to fix them yourself.
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
» Synchronizing configuration... ok
» Scheduling appointments... ok

> OpenJDK 11.0 Checkup...
  - 11.0.16.1 (C:\Program Files\Microsoft\jdk-11.0.16.101-hotspot\bin\..)

> Visual Studio 17.8.0 Checkup...
  - 17.8.6 - D:\Program Files\Microsoft Visual Studio\2022\Community

> Visual Studio Workloads Checkup...
  - .NET desktop development is installed (17.8.6)
  - Mobile development with .NET is installed (17.8.6)
  - ASP.NET and web development is installed (17.8.6)

> Android SDK Checkup...
  - emulator (33.1.24)
  - build-tools;34.0.0 (34.0.0)
  - platforms;android-34 (2)
  - system-images;android-34;google_apis;x86_64 (12)
  - platform-tools (34.0.5)
  - cmdline-tools;5.0 (5.0)

> Android Emulator Checkup...
  - Emulator: pixel_5_-_api_31_1 found.

> .NET SDK Checkup...
  - 3.1.426 - C:\Program Files\dotnet\sdk\3.1.426
  - 5.0.214 - C:\Program Files\dotnet\sdk\5.0.214
  - 5.0.408 - C:\Program Files\dotnet\sdk\5.0.408
  - 6.0.202 - C:\Program Files\dotnet\sdk\6.0.202
  - 6.0.203 - C:\Program Files\dotnet\sdk\6.0.203
  - 6.0.321 - C:\Program Files\dotnet\sdk\6.0.321
  - 7.0.115 - C:\Program Files\dotnet\sdk\7.0.115
  - 7.0.312 - C:\Program Files\dotnet\sdk\7.0.312
  - 8.0.101 - C:\Program Files\dotnet\sdk\8.0.101

> Windows Python Installation Checkup Checkup...
  - Python is installed in C:\Users\rafae\AppData\Local\Microsoft\WindowsApps\python.exe
.

> Windows Long Path Checkup Checkup...
  - Long paths are enabled on Windows!

> Windows Hyper-V Checkup Checkup...
  - Hyper-V is configured

> dotnet new Uno Project Templates Checkup...

> Edge WebView2 Checkup...
  - Found Edge WebView2 version 120.0.2210.144

> .NET SDK - Workloads (8.0.101) Checkup...
  - android (Microsoft.NET.Sdk.Android.Manifest-8.0.100 : 34.0.43/8.0.100) installed.
  - ios (Microsoft.NET.Sdk.iOS.Manifest-8.0.100 : 17.0.8478/8.0.100) installed.
  - maccatalyst (Microsoft.NET.Sdk.MacCatalyst.Manifest-8.0.100 : 17.0.8478/8.0.100) installed.
  - macos (Microsoft.NET.Sdk.macOS.Manifest-8.0.100 : 14.0.8478/8.0.100) installed.
  - maui (Microsoft.NET.Sdk.Maui.Manifest-8.0.100 : 8.0.3/8.0.100) installed.
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

- Congratulations, everything looks great!

Press Enter to finish...

**Reply by jeromelaban** [reply]
Thanks. Can you also provide a [binlog ](https://platform.uno/docs/articles/uno-builds-troubleshooting.html) of your build?

**Reply by Rafucho25** [reply]
Hi, sorry for the delay but i found part of the problem: i think it's the uno platform extension form visual studio, let me explain: every project i created with visual studio have this error so i created a new project with the Live Wizard for Rider & VS Code and surprice it works great so i decided to compare the nuget packages of both projects and i found that the visual studio extension(the project with errors) have 2 packages with different version than the project with live wizard. These are:

Package | Visual studio | Live wizard
| -------- | ------- | ------- |
| Uno.Resizetizer | 1.2.1 | 1.2.0 |
| Uno.Toolkit.WinUI | 5.1.5 | 5.1.3 |

i just downgrade the visual studio project to make equal to the live wizard and it works. If is useful this is the command created by live wizard:
`dotnet new unoapp -o testweb -preset "recommended" -id "com.companyname.testweb" -pub "O=testweb" -platforms "android" -platforms "wasm" -platforms "windows" -platforms "gtk" -theme-service  -skip  -theme "fluent" -presentation "mvux" -config  -di  -log "default" -nav "regions" -http False -loc  -server False -tests "none" -toolkit  -dsp False` 
and the version of the uno platform extension is: 5.1.22.24

**Reply by jeromelaban** [reply]
Thanks for the update. We're still not able to reproduce even with the different versions, can you provide a binlog of failing build?

**Reply by Rafucho25** [reply]
https://estuasdedu-my.sharepoint.com/:f:/g/personal/100401245_est_uasd_edu_do/EhbW4u7_TjZGh5Vye4TOnO8B4-t50HuERO_w-uCh7r9HEA?e=NoSljW This folder contains the binlog file, the project and other files.

**Reply by pictos** [reply]
@Rafucho25 I tried twice and couldn't download it... Not sure if it's my internet or the server. Right now everything has more than 200mb, Could you remove the obj/bin folders to reduce the size of the folder or push it to a git repository? Thanks in advance.

**Reply by jeromelaban** [reply]
It's best to attach the sources here (repositories may come and go) [using these steps](https://platform.uno/docs/articles/uno-howto-create-a-repro.html?tabs=windows-vs).

**Comment by Rafucho25** [comment]
Hi [test.zip](https://github.com/unoplatform/uno/files/14209137/test.zip)

**Reply by jeromelaban** [reply]
Let's track in https://github.com/unoplatform/uno/issues/15405
